### PR TITLE
Merge from dev to master

### PR DESCRIPTION
**Pull request by omcdanie**
Make smaller commits more often.

### CODE REVIEW

File: client/IndexComponent.js [hunk 1]
```diff
@@ -11,6 +13,7 @@ export default class Index extends Component {
             <BrowserRouter>
                 <Switch>
                     <Route exact path='/' component={ProfileView} />
+                    {/*<Route exact path='/' component={Bucket} />*/}
```
Comment (alredi) [inline client/IndexComponent.js]: We shouldn't be pushing commented out code

File: client/IndexComponent.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import React, {Component} from 'react';
 import {Route, BrowserRouter, Switch} from 'react-router-dom';
 
 import Login from '../imports/components/LoginComponent';
+import Bucket from '../imports/ui/buckets/BucketView';
```
Comment (alredi) [inline client/IndexComponent.js]: Let's try to be consistent - here we're importing "Bucket" and below "ProfileView", where the original files both have "View" in the title.

File: client/Main.css [hunk 3]
```diff
@@ -1,21 +1,27 @@
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
+
 html,
```
Comment (alredi) [inline client/Main.css]: This shows how to apply a global font so it doesn't need repeated for every item:

https://stackoverflow.com/questions/7025756/how-to-apply-global-font-to-whole-html-document

File: imports/ui/profile/BiographyComponent.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        return (
+            <section className='mini-bio-section'>
+                <section className="bio-container">
+                    <p>
+                        This is where the biography will be. There shall
+                        be an 'Edit' button in the bottom right hand of
+                        this component that will allow the user to
+                        update and change her/his biography at any time.
+                    </p>
+                    {/*<Label*/}
```
Comment (alredi) [inline imports/ui/profile/BiographyComponent.js:41]: More commented out code - this shouldn't be pushed.

**Review by alredi — APPROVED** [review]
made a few comments, but none of them are major.

File: client/IndexComponent.js (hunk 2 above)
Comment (omcdanie) [reply client/IndexComponent.js]: I hear what you're saying here, but keep in mind these imports are used for testing components until routing is implemented.

**Pull request merged**